client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -1,7 +1,130 @@
+const backupEventNames: StatusEventNames = {
+	inactive: {
+		small_screen: 'calypso_jetpack_agency_dashboard_add_backup_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_add_backup_click_large_screen',
+	},
+	progress: {
+		small_screen: 'calypso_jetpack_agency_dashboard_backup_progress_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_backup_progress_click_large_screen',
+	},
+	failed: {
+		small_screen: 'calypso_jetpack_agency_dashboard_backup_failed_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_backup_failed_click_large_screen',
+	},
+	warning: {
+		small_screen: 'calypso_jetpack_agency_dashboard_backup_warning_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_backup_warning_click_large_screen',
+	},
+	success: {
+		small_screen: 'calypso_jetpack_agency_dashboard_backup_success_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_backup_success_click_large_screen',
+	},
+};
+
+// Scan feature status event names for large screen(>960px) and small screen(<960px)
+const scanEventNames: StatusEventNames = {
+	inactive: {
+		small_screen: 'calypso_jetpack_agency_dashboard_add_scan_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_add_scan_click_large_screen',
+	},
+	progress: {
+		small_screen: 'calypso_jetpack_agency_dashboard_scan_progress_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_scan_progress_click_large_screen',
+	},
+	failed: {
+		small_screen: 'calypso_jetpack_agency_dashboard_scan_threats_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_scan_threats_click_large_screen',
+	},
+	success: {
+		small_screen: 'calypso_jetpack_agency_dashboard_scan_success_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_scan_success_click_large_screen',
+	},
+};
+
+// Montitor feature status event names for large screen(>960px) and small screen(<960px)
+const monitorEventNames: StatusEventNames = {
+	failed: {
+		small_screen: 'calypso_jetpack_agency_dashboard_monitor_site_down_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_monitor_site_down_click_large_screen',
+	},
+};
+
+// Plugin updates status event names for large screen(>960px) and small screen(<960px)
+const pluginEventNames: StatusEventNames = {
+	warning: {
+		small_screen: 'calypso_jetpack_agency_dashboard_update_plugins_click_small_screen',
+		large_screen: 'calypso_jetpack_agency_dashboard_update_plugins_click_large_screen',
+	},
+};
+
+// Returns event name needed for all the feature state clicks on the agency dashboard
+const getRowEventName = (
+	type: AllowedTypes,
+	status: AllowedStatusTypes,
+	isLargeScreen: boolean
+) => {
+	let eventName = '';
+	const deviceKey = isLargeScreen ? 'large_screen' : 'small_screen';
+	switch ( type ) {
+		case 'backup': {
+			eventName = backupEventNames?.[ status ]?.[ deviceKey ];
+			break;
+		}
```

Comment:
Since we're returning a single value here, IMO we can skip the declaration of `eventName`, but return the value directly.

```
// ...
case 'scan': {
    return scanEventNames?.[status]?.[deviceKey];
}
case 'monitor': {
    return monitorEventNames?.[status]?.[deviceKey];
}
// ...
```